examples/js/postprocessing/SAOPass.js

```diff
@@ -54,7 +54,7 @@ THREE.SAOPass = function ( scene, camera, depthTexture, useNormals, resolution )
 		var depthTexture = new THREE.DepthTexture();
 		depthTexture.type = THREE.UnsignedShortType;
 		depthTexture.minFilter = THREE.NearestFilter;
-		depthTexture.maxFilter = THREE.NearestFilter;
```

Comment:
Do you mind deleting both lines? `NearestFilter` is already the default value. There is no need to set.

Reply:
OK.